Makefile

```diff
@@ -8,8 +8,7 @@ clean:
 		rm -fr *.egg-info
 
 install:
-		pip install -e .
-		pip install -r dev_requirements.txt
+		pip install -e .[test]
```

Comment:
I assume this is `pip` syntax? Would it be safer to quote it (to prevent free regex globbing).